README.md

```diff
@@ -1,5 +1,13 @@
 # CRA MB Shared UI
 
+This template will get you started with all features in CRA as well as:
```

Comment:
Copy suggestion (with link added):

all [create-react-app](https://reactjs.org/docs/create-a-new-react-app.html) features

Reply:
Updated, thanks! I used the official CRA site linking to the getting started docs: https://create-react-app.dev/docs/getting-started